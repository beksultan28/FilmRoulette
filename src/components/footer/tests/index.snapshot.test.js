import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import FooterComponent from '../index.jsx';

test('FooterComponent renders correctly', () => {
  const footer = renderer.create(
    <MemoryRouter>
      <FooterComponent/>
    </MemoryRouter>,
  );
  expect(footer.toJSON()).toMatchSnapshot();
});
