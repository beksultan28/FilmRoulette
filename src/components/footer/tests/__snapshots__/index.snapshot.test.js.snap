// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent renders correctly 1`] = `
<div
  className="footer"
>
  <div
    className="container"
  >
    <a
      className="logo"
      href="/"
      onClick={[Function]}
    >
      FilmRoulette
    </a>
  </div>
</div>
`;
